components/visualping/sources/new-job-event/new-job-created.mjs

```diff
@@ -0,0 +1,43 @@
+import visualping from "../../app/visualping.app.mjs";
+import { DEFAULT_POLLING_SOURCE_TIMER_INTERVAL } from "@pipedream/platform";
+
+export default {
+  name: "New Job Event",
+  version: "0.0.1",
+  key: "visualping-new-job-event",
+  description: "Emit new event for each new job event.",
```

Comment:
```suggestion
  description: "Emit a new event for each new job event.",
```

Reply:
Changed "Emit new event" to "Emit a new event" (Google's Developer docs style guide)